sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -180,6 +180,14 @@ func run() (err error) {
 		return fmt.Errorf("static analysis error: %w", err)
 	}
 
+	startHashTime := time.Now()
+	archiveHash, err := utils.SHA256Hash(archivePath)
+	if err != nil {
+		slog.WarnContext(ctx, "failed to calculate archive checksum", "error", fmt.Sprintf("%s", err))
```

Comment:
nit: `fmt.Sprintf` is unnecessary, you can just write `err` and it will be stringified by the library :)

Reply:
Done !! Thanks :)